WordPress/gutenberg
# Storybook: Add stories for AlignmentToolbar and AlignmentControl components

## What?

This PR will add stories for AlignmentToolbar and AlignmentControl components in the Storybook

## Why?

Most of Block Editor components don't have stories.
Block Editor components's Storybook should also be added and updated like the Components.
see #66519 
related #22891 

## How?

I have referred to the TextAlignmentControl story.
AlignmentControl and AlignmentToolbar are based on same component, but there are deliberately generated as a component with a different name, so I created in two separate files.

## Testing Instructions




`npm run storybook:dev` and please check the AlignmentControl and AlignmentToolbar stories.

### Testing Instructions for Keyboard


## Screenshots or screencast 

https://github.com/user-attachments/assets/524be1c3-90b1-4b0b-8906-0e92cbdbac35
